packages/gatsby/test/fixtures/gatsby-v2/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "gatsby-v2",
```

Comment:
@lydiahallie can you help me with the v2 test fixture? I've removed everything I can think of but am still seeing this error:

```
test/fixtures/gatsby-v2 on  add/vercel-gatsby-plugin [$✘!?] via 🤖 v16.18.1 on ☁️  production took 4s 
❯ yarn develop
yarn run v1.22.19
$ gatsby develop
/Users/ethanarrowood/Documents/github/vercel/vercel/packages/gatsby/test/fixtures/gatsby-v2/node_modules/gatsby-recipes/dist/graphql-server/server.j
s:52
var remarkMdx = require('remark-mdx');
                ^

Error [ERR_REQUIRE_ESM]: require() of ES Module
/Users/ethanarrowood/Documents/github/vercel/vercel/packages/gatsby/test/fixtures/gatsby-v2/node_modules/remark-mdx/index.js from /Users/ethanarrowo
od/Documents/github/vercel/vercel/packages/gatsby/test/fixtures/gatsby-v2/node_modules/gatsby-recipes/dist/graphql-server/server.js not supported.
Instead change the require of index.js in /Users/ethanarrowood/Documents/github/vercel/vercel/packages/gatsby/test/fixtures/gatsby-v2/node_modules/g
atsby-recipes/dist/graphql-server/server.js to a dynamic import() which is available in all CommonJS modules.
    at Object.<anonymous> (/Users/ethanarrowood/Documents/github/vercel/vercel/packages/gatsby/test/fixtures/gatsby-v2/node_modules/gatsby-recipes/d
ist/graphql-server/server.js:52:17) {
  code: 'ERR_REQUIRE_ESM'
}


success open and validate gatsby-configs - 0.059s
success load plugins - 0.028s
success onPreInit - 0.012s
success initialize cache - 0.005s
success copy gatsby files - 0.063s
success onPreBootstrap - 0.010s
success createSchemaCustomization - 0.002s
success Checking for changed pages - 0.001s
success source and transform nodes - 0.017s
success building schema - 0.073s
info Total nodes: 18, SitePage nodes: 1 (use --verbose for breakdown)
success createPages - 0.001s
success Checking for changed pages - 0.000s
success createPagesStatefully - 0.026s
success update schema - 0.009s
success write out redirect data - 0.001s
success onPostBootstrap - 0.001s
info bootstrap finished - 1.694s
success onPreExtractQueries - 0.001s
success extract queries from components - 0.061s
success write out requires - 0.011s
success run static queries - 0.025s - 1/1 40.27/s
success run page queries - 0.012s - 1/1 85.05/s
⠀
You can now view gatsby-v2 in the browser.
⠀
  http://localhost:8000/
⠀
View GraphiQL, an in-browser IDE, to explore your site's data and schema
⠀
  http://localhost:8000/___graphql
⠀
Note that the development build is not optimized.
To create a production build, use gatsby build
⠀
success Building development bundle - 2.654s
success onPreExtractQueries - 0.001s
success extract queries from components - 0.026s
success write out requires - 0.001s
success Re-building development bundle - 0.040s
```

Yet as you can see, it all still works. Not sure if that matters? Just trying to get a simple v2 test case